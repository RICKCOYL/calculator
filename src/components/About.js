/* eslint-disable max-len */
import React from 'react';

const About = () => (
  <div>
    <h1>About</h1>
    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit ut ex aperiam esse iusto porro quos error libero adipisci aspernatur sapiente nam, possimus quae commodi, dolore atque est consectetur repellat.</p>
  </div>
);

export default About;
