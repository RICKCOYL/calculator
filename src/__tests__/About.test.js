import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '../components/About';

describe('Home component test', () => {
  test('should renders welcome text', () => {
    render(<About />);
    const text = screen.getByText('About');
    expect(text).toBeInTheDocument();
  });

  test('should render the expected output', () => {
    const component = renderer.create(
      <About />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
