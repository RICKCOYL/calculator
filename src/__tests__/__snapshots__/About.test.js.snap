// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test should render the expected output 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit ut ex aperiam esse iusto porro quos error libero adipisci aspernatur sapiente nam, possimus quae commodi, dolore atque est consectetur repellat.
  </p>
</div>
`;
